### DISCUSSION TITLE
Test Intel Xeon Phi Card

### DISCUSSION THREAD

**Discussion by tsukisan** [geerlingguy/raspberry-pi-pcie-devices#198] [Ideas]
I'm curious if (some of) the power of a Phi card could be utilized by the Pi:

https://www.amazon.com/Xeon-Phi-5110P-1-05-Coprocessor/dp/B00FF5AP1O/

**Comment by geerlingguy** [comment]
(Just noting I moved this to a discussion since I'm likely not going to test this on my own. A card like this is very likely tied into the X86 ecosystem so much that it wouldn't work with ARM.)
